rust/serving/src/config.rs

```diff
@@ -11,7 +11,8 @@ use std::collections::HashMap;
 use std::fmt::Debug;
 use tracing::info;
 
-const ENV_NUMAFLOW_SERVING_APP_PORT: &str = "NUMAFLOW_SERVING_APP_LISTEN_PORT";
+const ENV_NUMAFLOW_SERVING_APP_HTTPS_PORT: &str = "NUMAFLOW_SERVING_APP_LISTEN_PORT";
+pub const ENV_NUMAFLOW_SERVING_APP_HTTP_PORT: &str = "NUMAFLOW_SERVING_APP_LISTEN_HTTP_PORT";
```

Comment:
```suggestion
const ENV_NUMAFLOW_SERVING_APP_HTTPS_PORT: &str = "NUMAFLOW_SERVING_APP_LISTEN_HTTPS_PORT";
pub const ENV_NUMAFLOW_SERVING_APP_HTTP_PORT: &str = "NUMAFLOW_SERVING_APP_LISTEN_HTTP_PORT";
```

Reply:
`NUMAFLOW_SERVING_APP_LISTEN_PORT` this is used elsewhere in go code and hence I didnt change